connectors/atp/requirements.txt

```diff
@@ -1,2 +1,2 @@
 adal==1.2.2
-car-connector-framework==3.0.1
+car-connector-framework==3.0.2rc269
```

Comment:
are you going to merge this PR after we have a prod version of framework? say 3.0.3

Reply:
Replaced with 4.0.0